lib/test/karma.js

```diff
@@ -166,43 +179,25 @@ function karmaRun(reporters, bannerWidth) {
     var data    = querystring.escape(JSON.stringify(options));
     var command = [ 'node', quote(appPath), data ].join(' ');
 
-    childProcess.exec(command, { cwd: process.cwd() }, function reporter(stderr, stdout) {
-      if (reporters.length > 0) {
-        if (stdout) {
-          process.stdout.write(stdout);
-        }
-        if (stderr) {
-          process.stderr.write(stderr);
-        }
-        done();
-        return;
-      }
-
-      //TODO @bguiz replace reporter function with a standard karma reporter,
-      //and extract it to own module
-
-      //default reporter by parsing the stdout and stderr of karma
-      var report;
+    //NOTE this workaround is necessary, see issue:
+    //https://github.com/sindresorhus/supports-color/issues/13
+    //TODO @bguiz remove workaround when issue has been resolved
+    childProcess.exec(command, {
+      cwd: process.cwd(),
+      env: lodashMerge(process.env, {
+        SUPPORTS_COLOR: true,
+      }),
+    }, function reporter(err, stdout, stderr) {
+      //NOTE it is safe to ignore the error,
+      //because karma-runner exits with a non-zero exit code when any tests fail.
       if (stdout) {
```

Comment:
I think there is an option in `childProcess.exec` or a variant by which you do not intercept `stdout` and `stderr`.
This would now be more appropriate since we are no longer processing their content.